### PR TITLE
bf(S3C-3263): Fix objectDelta field values

### PR DESCRIPTION
**Pull request by tmacro**
# Pull request template

### Motivation and context

Why is this change required? What problem does it solve?

### Related issues

Please use the following link syntaxes #600 to reference issues in the
current repository

### Add tests to cover the changes

New tests added or existing tests modified to cover all changes

### Sign your work

In order to contribute to the project, you must sign your work
https://github.com/scality/Guidelines/blob/master/CONTRIBUTING.md#sign-your-work

Thank you again for contributing! We will try to test and integrate the change
as soon as we can.

### CODE REVIEW

File: lib/api/multipartDelete.js [hunk 1]
```diff
@@ -51,6 +51,8 @@ function multipartDelete(authInfo, request, log, callback) {
                     keys: [objectKey],
                     byteLength: partSizeSum,
                     location,
+                    numberOfObjects: 1,
```
Comment (rahulreddy) [inline lib/api/multipartDelete.js]: We don't have to set `numberOfObjects` here as it's not counted until mpu is finished

File: lib/api/objectPutPart.js [hunk 2]
```diff
@@ -374,6 +374,7 @@ function objectPutPart(authInfo, request, streamingV4Params, log,
             newByteLength: size,
             oldByteLength: prevObjectSize,
             location: destinationBucket.getLocationConstraint(),
+            numberOfObjects: 1,
```
Comment (rahulreddy) [inline lib/api/objectPutPart.js]: Same here, we don't need to increment `numberOfObjects` here. It gets incremented in complete mpu api.

**Review by rahulreddy — CHANGES_REQUESTED** [review]

**Review by jonathan-gramain — APPROVED** [review]

**Review by rahulreddy — APPROVED** [review]

**Pull request merged**